Classes/ViewHelpers/Format/SubStringLastViewHelper.php

```diff
@@ -1,27 +1,28 @@
 <?php
 namespace Slub\Dfgviewer\ViewHelpers\Format;
-/***************************************************************
- *  Copyright notice
+
+/**
+ * Copyright notice
  *
- *  (c) 2019 Alexander Bigga <typo3@slub-dresden.de>
- *  All rights reserved
+ * (c) Saxon State and University Library Dresden <typo3@slub-dresden.de>
+ * All rights reserved
  *
- *  This script is part of the TYPO3 project. The TYPO3 project is
- *  free software; you can redistribute it and/or modify
- *  it under the terms of the GNU General Public License as published by
- *  the Free Software Foundation; either version 3 of the License, or
- *  (at your option) any later version.
+ * This script is part of the TYPO3 project. The TYPO3 project is
+ * free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
```

Comment:
> In the files there is no change from 3 to 2.

Is this here (and several more) no change from 3 to 2?

Reply:
Oh, I see, it looks that part of the files was with 3 and part with 2. I have adjusted in one file and then copy it over. So even one more reason to unify those files.

Thanks for the catch. I will switch all the files to 3.